app/root.tsx

```diff
@@ -10,9 +11,12 @@ export function Layout({ children }: { children: React.ReactNode }) {
         <Meta />
         <Links />
       </head>
-      <body className={styles.root}>
-        {children}
-        <Scripts />
+      <body className={styles.container}>
+        <SideBar />
+        <div className={styles.root}>
+          {children}
+          <Scripts />
+        </div>
```

Comment:
```suggestion
      <body>
        <div className={styles.center}>
          <div className={styles.layout}>
            <SideBar />
            <div className={styles.root}>
              {children}
              <Scripts />
            </div>
          </div>
        </div>
```
そろそろ `body` に直接スタイルを当てるのは厳しくなってきていそうです. 中央寄せする要素と画面要素を配置する要素を分けたうえで, 次のようなスタイルを付けてみてはどうでしょう:
```css
body {
  margin: 0;
}

.center {
  display: flex;
  justify-content: center;
}

.layout {
  display: grid;
  grid-template-columns: min-content 1fr;
}
```